pkg/ovs/openflow/ofctrl_nxfields.go

```diff
@@ -70,10 +70,6 @@ func (f *XXRegField) GetNXFieldName() string {
 	return fmt.Sprintf("%s%d", NxmFieldXXReg, f.regID)
 }
 
-func (f *XXRegField) isFullRange() bool {
```

Comment:
Should we remove this or skip unused lint in this file? @wenyingd @tnqn

Reply:
I don't why this was added, will defer to @wenyingd

Reply:
type XXRegField is an alias of RegField. It is used for IPv6. I check the existing code, IPv6 case doesn't needs the removed the function.